catalogue/src/main/kotlin/com/tidal/sdk/catalogue/generated/infrastructure/StringBuilderAdapter.kt

```diff
@@ -0,0 +1,20 @@
+package com.tidal.sdk.catalogue.generated.infrastructure
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializer
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.SerialDescriptor
+
+@Serializer(forClass = StringBuilder::class)
+object StringBuilderAdapter : KSerializer<StringBuilder> {
```

Comment:
I'm not sure why we need to deserialize to a StringBuilder tbh since this object is usually a tool to build a full string. Couldn't find the usage as well